e2e/playwright/point-click.spec.ts

```diff
@@ -3628,4 +3628,540 @@ solid001 = extrude(sketch001, length = 5)`
       await editor.expectEditor.not.toContain('patternLinear3d(')
     })
   })
+
+  test('GDT Flatness from command bar', async ({
+    context,
+    page,
+    homePage,
+    scene,
+    editor,
+    toolbar,
+    cmdBar,
+  }) => {
+    const initialCode = `@settings(defaultLengthUnit = in, experimentalFeatures = allow)
+sketch001 = startSketchOn(XZ)
+  |> circle(center = [0, 0], radius = 30)
+extrude001 = extrude(sketch001, length = 30)
+    `
+    await test.step('Settle the scene', async () => {
+      await context.addInitScript((initialCode) => {
+        localStorage.setItem('persistCode', initialCode)
+      }, initialCode)
+      await page.setBodyDimensions({ width: 1000, height: 500 })
+      await homePage.goToModelingScene()
+      await scene.settled(cmdBar)
+
+      // Close panes to ensure toolbar buttons are visible
+      await toolbar.closePane('feature-tree')
+      await toolbar.closePane('code')
+    })
+
+    // Adjusted coordinates to center screen for clicking on cap
+    const testPoint = { x: 500, y: 200 }
+    const [clickOnCap] = scene.makeMouseHelpers(testPoint.x, testPoint.y)
```

Comment:
I think we could click on a wall instead? Or something that we could select in code instead of abs px coordinates. Not a blocker though, could be a follow up

Reply:
I swear I was able to do this in the past, after posting that comment I went back and couldn't find examples, sorry